Make sure only qmake is built, as the rest of Qt is built
in other ports.

--- configure.orig	2016-08-24 08:18:37 UTC
+++ configure
@@ -4115,11 +4115,11 @@ if [ '!' -e "$outpath/bin/qmake" ]; then
         echo "QT_MAJOR_VERSION = $QT_MAJOR_VERSION" >> "$mkfile"
         echo "QT_MINOR_VERSION = $QT_MINOR_VERSION" >> "$mkfile"
         echo "QT_PATCH_VERSION = $QT_PATCH_VERSION" >> "$mkfile"
-        echo "EXTRA_CFLAGS = $EXTRA_CFLAGS" >> "$mkfile"
-        echo "EXTRA_CXXFLAGS = $EXTRA_CXXFLAGS" >> "$mkfile"
+        echo "EXTRA_CFLAGS = $EXTRA_CFLAGS $CFLAGS" >> "$mkfile"
+        echo "EXTRA_CXXFLAGS = $EXTRA_CXXFLAGS $CXXFLAGS" >> "$mkfile"
         echo "QTOBJS =" $EXTRA_OBJS >> "$mkfile"
         echo "QTSRCS =" $EXTRA_SRCS >> "$mkfile"
-        echo "LFLAGS = $EXTRA_LFLAGS" >> "$mkfile"
+        echo "LFLAGS = $EXTRA_LFLAGS $LDFLAGS" >> "$mkfile"
         echo "EXEEXT = $EXEEXT" >> "$mkfile"
         echo "RM_F = rm -f" >> "$mkfile"
         echo "RM_RF = rm -rf" >> "$mkfile"
@@ -4138,6 +4138,7 @@ if [ '!' -e "$outpath/bin/qmake" ]; then
             fi
         fi
     done
+    exit 0
 
     if [ "$OPT_VERBOSE" = yes ]; then
         # Show the output of make
